packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -33,7 +33,12 @@ export type AsyncTask = Task & {
 };
 
 export class InteractionTracker {
+  // Allows to track several events triggered by the same user interaction in the right Zone.
+  private shouldCreateNewZone = true;
+  private currentZone: Zone = Zone.root;
```

Comment:
What would you think about calling this something like `currentEventTracingZone` or similar? When I see `currentZone` it makes me wonder how that is different from `Zone.current`.